crates/net/src/sim2h_worker.rs

```diff
@@ -333,6 +335,42 @@ impl Sim2hWorker {
         }
     }
 
+    fn self_store_authored_aspects(&mut self) {
```

Comment:
The changes in this file make the booting of a conductor faster in the case of already holding/authoring a lot. With yesterday's naive implementation we would always send store (and first fetch-) requests for everything in the authoring list, no matter if we are holding already. This waits until we have both lists so we can actually compare if there is anything in the source chain that needs to be held.
(Which needs to be here in sim2h because it is a full-sync DHT assumption and should go away or get deactivated once we activate sharding)

Reply:
If there is code here that needs to go away can you make that visible in the comments with a TODO?

Reply:
@maackle @lucksus i don't think this is worth blocking a merge here as we are getting fairly overdue for a release and @lucksus is heading off for the year...

a TODO would be nice but also it sounds like sharding simply won't work until this is upgraded so i doubt we'd miss it :man_shrugging:

Reply:
To me it sounds like sharding will work just fine, but we'll be potentially Holding stuff that the sharded network doesn't need us to hold. Either way I guess it's enough that at least the three of us have taken note.